### PR TITLE
feat(styles): add new Time Picker component

### PR DESCRIPTION
**Pull request by InnaAtanasova**
## Related Issue
Closes https://github.com/SAP/fundamental-styles/issues/3868

## Description
- deprecated the Pattern Time Picker
- added a new component Time Picker
- added a modifier class for buttons when used in Dialog footer `fd-button--decisive`

### CODE REVIEW

File: packages/styles/src/timepicker.scss [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+@while $i < 361 {
+  .#{$block}__item--deg#{$i} {
+    transform: translate(-50%) rotate(#{$i}deg);
+
+    .#{$block}__number {
+      transform: rotate(-#{$i}deg);
+    }
+  }
+  $i: $i + 6;
+}
```
Comment (g-cheishvili) [inline packages/styles/src/timepicker.scss:188]: CSS variables would take a lot less space for this one

Comment (g-cheishvili) [reply packages/styles/src/timepicker.scss:188]: But I would go with [Data attributes](https://css-tricks.com/a-complete-guide-to-data-attributes/#aa-using-data-attributes-visually)

**Review by g-cheishvili — CHANGES_REQUESTED** [review]

File: packages/styles/src/timepicker.scss (hunk 1 above)
Comment (InnaAtanasova) [reply packages/styles/src/timepicker.scss:188]: could you be more specific?

**Comment by InnaAtanasova** [comment]
@g-cheishvili your comments have been addressed. Please check again. Thanks

**Comment by g-cheishvili** [comment]
> @g-cheishvili your comments have been addressed. Please check again. Thanks

```12:29:00 AM: Could not resolve '../../../src/popover.scss' from packages/styles/stories/Deprecated/Components/time-picker/time-picker.stories.js```
It is not building
Also I could not see in HTML how degrees are passed

**Review by g-cheishvili — APPROVED** [review]

**Pull request merged**